src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -223,6 +233,32 @@ export default class PhotoField extends React.Component {
     return this.state.errorMessage;
   }
 
+  getCanvasCropBoundariesMet = (canvasData, cropboxData) => {
+    const { height: canvasHeight, width: canvasWidth, left: canvasLeft, top: canvasTop } = canvasData;
+    const { height: cropboxHeight, width: cropboxWidth, left: cropboxLeft, top: cropboxTop } = cropboxData;
+
+    // origin is at center
+    // coodinates are for canvas's container
+    const canvasBoundaries = {
+      minLeft: Math.round(-canvasWidth + cropboxLeft + cropboxWidth, 2),
+      maxLeft: Math.round(cropboxLeft, 2),
+      minTop: Math.round(-canvasHeight + cropboxTop + cropboxHeight, 2),
+      maxTop: Math.round(cropboxTop, 2),
+    };
+
+    return {
+      bottomBoundaryMet: Math.round(canvasTop, 2) === canvasBoundaries.minTop,
+      topBoundaryMet: Math.round(canvasTop, 2) === canvasBoundaries.maxTop,
+      leftBoundaryMet: Math.round(canvasLeft, 2) === canvasBoundaries.maxLeft,
+      rightBoundaryMet: Math.round(canvasLeft, 2) === canvasBoundaries.minLeft
```

Comment:
Updated some terms here and broke apart the boundary determination from the UI updating.